### ISSUE TITLE
Embedding Docs

### ISSUE THREAD

**Issue by tmbo** [RasaHQ/rasa_core#713]

**Comment by Ghostvv** [comment]
Finally, understood what's wrong with skip gate: it multiplies hidden memory with a number from 0 to 1, causing vanishing gradients. So needed to refactor this behavior. The new version is working, need to clean it up, and make different flags working for ablation tests

**Comment by Ghostvv** [comment]
next: start ablation tests

**Comment by amn41** [comment]
I wouldn't (for now) delay this again to do more ablation tests. Let's get a working version into rasa core and then we can still iterate on it

**Comment by Ghostvv** [comment]
Agree, ablation tests are just more for actual testing if several flags (on and off)  are working together without exception errors.

**Comment by amn41** [comment]
ah ok understood :)

**Comment by Ghostvv** [comment]
Update: @tmbo reviewed the code. I'll refactor it according to his comments and add explanatory comments why certain thing were included. These comments will create the basis for the embed policy docs.

**Comment by Ghostvv** [comment]
The refactor is ready! Next stop a new scheme of the policy for the docs
